src/components/main-page/FAQ/FAQDisclosure/FAQDisclosure.module.scss

```diff
@@ -0,0 +1,72 @@
+.buttonBackground {
+  background-color: var(--color-white);
+  border-radius: var(--border-radius-default) var(--border-radius-default) 0 0;
+  transition: background-color 0.3s linear;
+}
+
+.button {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  text-align: left;
+  font-family: inherit;
+  font-size: var(--font-size-m);
+  padding: var(--space-s) var(--space-m);
+  background-color: var(--color-blue-light);
+  color: var(--color-white);
+  border: none;
+  border-radius: var(--border-radius-default);
+  cursor: pointer;
+  width: 100%;
+}
+
+.chevron {
+  height: var(--chevron-height);
+  fill: var(--color-white);
+  transform: rotate(0);
+  transition: transform 0.5s ease-out;
+
+  &.open {
+    transform: rotate(180deg);
+  }
+}
+
+.panel {
+  background-color: var(--color-white);
+  color: var(--color-black);
+  padding: var(--space-m);
+  border-radius: 0 0 var(--border-radius-default) var(--border-radius-default);
+  animation-delay: 0.2s;
+}
+
+.enter {
+  transition: all 0.2s ease-out;
```

Comment:
I'm wondering if it wouldn't look better if we'd increase the values a little bit - we could check out if with the values from `Navigation` it wouldn't look better, wdyt?